poseidon-hs.cabal

```diff
@@ -1,5 +1,5 @@
 name:                poseidon-hs
-version:             1.5.7.0
+version:             1.5.7.1
```

Comment:
I took v1.5.7.1 already for #313. Wer zuerst kommt patched zuerst :grimacing:

Reply:
OK true. I will take whatever comes next.